packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -133,13 +133,16 @@ export function isTheia(): boolean {
 
 export async function isUsingTokenAuth(profileName: string): Promise<boolean> {
     const config = (await Profiles.getInstance().getProfileInfo()).getTeamConfig();
-    const profile = await Profiles.getInstance().getLoadedProfConfig(profileName);
+    const defaultBaseProfileName = Profiles.getInstance().getDefaultProfile("base").name;
+    const baseProfile = config.properties.profiles[defaultBaseProfileName];
+    const profile = config.properties.profiles[profileName];
```

Comment:
This won't work to access the profile if it's a nested profile like `lpar1.zosmf`.

In order to check if a secure field exists for a profile, we could do it like this:
`config.api.secure.securePropsForProfile(profileName).includes("tokenValue")`

Reply:
Sorry for the confusing comment, as @zFernand0 mentioned at standup we should probably be using the ProfileInfo API instead to ensure that both v1 and v2 profiles are supported 🙂  I don't think the ProfileInfo API has a convenient method to check if a secure property exists for a profile, but we could add one to the Profiles class:
```typescript
    public async getSecurePropsForProfile(profileName: string): Promise<string[]> {
        const profAttrs = await this.getProfileFromConfig(profileName);
        const mergedArgs = (await this.getProfileInfo()).mergeArgsForProfile(profAttrs);
        return [...mergedArgs.knownArgs, ...mergedArgs.missingArgs].filter((arg) => arg.secure).map((arg) => arg.argName);
    }
```